ext/crypto/00_crypto.js

```diff
@@ -606,6 +657,76 @@
       }
     }
 
+    /**
+    * @param {AlgorithmIdentifier} algorithm
+    * @param {CryptoKey} baseKey
+    * @param {number} length
+    * @returns {Promise<ArrayBuffer>}
+    */
+    async deriveBits(algorithm, baseKey, length) {
+      webidl.assertBranded(this, SubtleCrypto);
+      const prefix = "Failed to execute 'deriveBits' on 'SubtleCrypto'";
+      webidl.requiredArguments(arguments.length, 3, { prefix });
+      algorithm = webidl.converters.AlgorithmIdentifier(algorithm, {
+        prefix,
+        context: "Argument 1",
+      });
+      baseKey = webidl.converters.CryptoKey(baseKey, {
+        prefix,
+        context: "Argument 2",
+      });
+      length = webidl.converters["unsigned long"](length, {
+        prefix,
+        context: "Argument 3",
+      });
+
+      const normalizedAlgorithm = normalizeAlgorithm(algorithm, "deriveBits");
+      switch (normalizedAlgorithm.name) {
+        case "PBKDF2": {
+          // 1.
+          if (length == null || length == 0 || length % 8 !== 0) {
+            throw new DOMException("Invalid length", "OperationError");
+          }
+
+          // TODO(@littledivy): Add this step to spec. WPT has tests for it.
```

Comment:
It is in spec: https://w3c.github.io/webcrypto/#pbkdf2-operations. Are you looking at the candidate recommendation rather than the editors draft?

Reply:
Forgot to remove that...added it to the spec myself (https://github.com/w3c/webcrypto/commit/d2917f41d1df1f8937718b67ee4746cf74515446)